cfme/tests/configure/test_start_page.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import fauxfactory
+
+from cfme.base.credential import Credential
+from cfme.configure.settings import Visual
+from cfme.configure.access_control import Group, User
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+faux_value = fauxfactory.gen_alphanumeric()
+pytestmark = pytest.mark.usefixtures('browser')
+
+
+def get_landing_page():
+    view = navigate_to(Visual, 'All')
+    landing_pages = view.visualstartpage.show_at_login.all_options
+    start_page = []
+    for landing_page in landing_pages:
+        start_page.append(landing_page.text)
+    return start_page
+
+
+@pytest.yield_fixture(scope="module")
+def rbac_user(appliance, role):
+    role = role
+    group = Group(description='test{}'.format(faux_value), role=role.name)
+    group.create()
+    group_user = Group('test{}'.format(faux_value))
```

Comment:
**REQUIRED:** Where is ```group``` used? We create ```group```, but then we pass ```group_user``` into the ```User``` object. We should be able to pass ```group``` to ```User```. Any reason not to?